lib/services/samples-service.ts

```diff
@@ -20,13 +20,16 @@ class Sample {
 
 export class SamplesService implements ISamplesService {
 	private static GITHUB_ICENIUM_LOCATION_ENDPOINT = "https://api.github.com/orgs/Icenium/repos?per_page=100";
-	private static GITHUB_TELERIK_LOCATION_ENDPOINT = "https://api.github.com/orgs/telerik/repos?per_page=100";
 	private static GITHUB_REGEX = /https:\/\/github[.]com\/Icenium\/(?!deprecated-)(sample-|.*?-sample-)[\w\W]+[.]git$/i;
 	private static NAME_FORMAT_REGEX = /(sample-|-)/gi;
 	private static NAME_PREFIX_REMOVAL_REGEX = /(sample-)/i;
 	private static REMOTE_LOCK_STATE_PRIVATE = "private";
 	private static SAMPLES_PULL_FAILED_MESSAGE = "Failed to retrieve samples list. Please try again a little bit later.";
 	private static NATIVESCRIPT_SAMPLE_CUTENESS_NAME = "nativescript-sample-cuteness";
+	private static GITHUB_CORDOVA_SAMPLES_REGEX = new RegExp("https:\/\/github[.]com\/Icenium\/(?!deprecated-)sample-[\\w\\W]+[.]git$", "i");
+	private static GITHUB_MOBILE_WEBSITE_SAMPLES_REGEX = new RegExp("https:\/\/github[.]com\/Icenium\/(?!deprecated-)mobilewebsite-sample-[\\w\\W]+[.]git$", "i");
+	private static GITHUB_NS_SAMPLES_REGEX = new RegExp("https:\/\/github[.]com\/Icenium\/(?!deprecated-)nativescript-sample-[\\w\\W]+[.]git$", "i");
```

Comment:
remove (?!deprecated-) part from this Regex and from all related except GITHUB_REGEX. samples starting with deprecated can never match this regex.